docs/content/docs/connectors/table/hbase.md

```diff
@@ -188,20 +188,48 @@ Connector Options
       <td>Whether async lookup are enabled. If true, the lookup will be async. Note, async only supports hbase-2.2 connector.</td>
     </tr>
     <tr>
-      <td><h5>lookup.cache.max-rows</h5></td>
+      <td><h5>lookup.cache</h5></td>
       <td>optional</td>
       <td>yes</td>
-      <td style="word-wrap: break-word;">-1</td>
+      <td style="word-wrap: break-word;">NONE</td>
+      <td><p>Enum</p>Possible values: NONE, PARTIAL</td>
+      <td>The cache strategy for the lookup table. Currently supports NONE (no caching) and PARTIAL (caching entries on lookup operation in external database).</td>
+    </tr>
+    <tr>
+      <td><h5>lookup.partial-cache.max-rows</h5></td>
```

Comment:
Refer to https://github.com/apache/flink/blob/bf81768ff564c5bf4a57cb33c6f5126b83b28fb5/flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/table/HBaseConnectorOptions.java#L98, I think we should also explain the old compatible options instead of dropping it directly. Other deprecated options are similar.

Reply:
I think keeping some deprecated options in the doc is a bit weird too, as we should "push" users to switch to new options asap. We keep these deprecated options in the code just for backward compatibility. Users can always refer to doc of previous versions to get the definition of those deprecated options.

Reply:
If users don't know the alternatives to old options, how do you push them to migrate to new options? Reading the code? This will be more expensive due to users can't easily get the old and new options mapping. Before we drop the old options in the code, I think we should make user can get the mapping easily, so the migration will be more smooth. Maybe refer to the `Deprecated Options [#](https://nightlies.apache.org/flink/flink-docs-master/zh/docs/deployment/config/#deprecated-options)`?

Reply:
Thanks for the feedback! Of course users don't need to read the code to get the mapping. Both deprecated and new options are descriptive enough in the doc to be translated between. Anyhow I get your point to make the migration more smooth. I'll add a new section to show which deprecated option the new one is mapping to.